### PR TITLE
Title in installation

### PR DESCRIPTION
**Pull request by jreidinger**
tested manually in VM

### CODE REVIEW

File: src/y2start/y2start [hunk 1]
```diff
@@ -49,7 +49,9 @@ hostname = " @ #{hostname}" unless hostname.empty?
 
 title = "YaST2 - #{args[:client_name]}#{hostname}"
 # set title only if it is not CLI (bsc#1033993)
-Yast::UI.SetApplicationTitle(title) if args[:client_options][:params].empty?
+# modules do not have CLI arguments, except installation (bsc#1037891)
+set_title = args[:client_options][:params].empty? || args[:client_name] == "installation"
```
Comment (imobachgs) [inline src/y2start/y2start]: So I guess that `args[:client_options]` and `args[:client_options][:params]` could not be `nil`, right?

Comment (jreidinger) [inline src/y2start/y2start]: yes, it is result of params parsing. Always defined.

Comment (jreidinger) [inline src/y2start/y2start]: see https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/y2start_helpers.rb#L12

Comment (imobachgs) [inline src/y2start/y2start]: Just to be sure, thanks!

**Pull request merged**